backtracepython/report_queue.py

```diff
@@ -8,9 +8,10 @@
 
 
 class ReportQueue:
-    def __init__(self, request_handler, source_code_handler=None):
+    def __init__(self, request_handler, exit_timeout=None, source_code_handler=None):
```

Comment:
Is this class internal? If not, it would be better to put `exit_timeout` after `source_code_handler` to not break the API.

Reply:
Yes it's internal